src/pytorch_lightning/loops/fit_loop.py

```diff
@@ -210,7 +210,8 @@ def on_run_start(self) -> None:  # type: ignore[override]
 
         self.trainer.reset_train_dataloader(self.trainer.lightning_module)
         # reload the evaluation dataloaders too for proper display in the progress bar
-        self.epoch_loop.val_loop._reload_evaluation_dataloaders()
+        if self.epoch_loop._should_check_val_epoch():
+            self.epoch_loop.val_loop._reload_evaluation_dataloaders()
```

Comment:
reload for tqdm only if validation is supposed to happen during that epoch, else don't